<ui version="4.0" >
 <class>GeneralWidget</class>
 <widget class="QWidget" name="GeneralWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>起動時の動作</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="updateAtStartupCheckBox" >
        <property name="text" >
         <string>イエローページからチャンネルを取得する</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>イエローページ</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="dontShowMinusScoreChannelCheckBox" >
        <property name="text" >
         <string>スコアがマイナスなチャンネルは表示しない</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoUpdateCheckBox" >
        <property name="text" >
         <string>自動更新をデフォルトにする</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>自動更新の間隔</string>
          </property>
          <property name="indent" >
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="autoUpdateIntervalSpinBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix" >
           <string>分</string>
          </property>
          <property name="minimum" >
           <number>3</number>
          </property>
          <property name="maximum" >
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>217</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>システムトレイ／通知エリア</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="showSystemTrayIconCheckBox" >
        <property name="text" >
         <string>アイコンを表示する</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>ウェブ・ブラウザ</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QRadioButton" name="useCommonWebBrowserRadioButton" >
        <property name="text" >
         <string>デスクトップ環境の既定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="useCustomWebBrowserRadioButton" >
        <property name="text" >
         <string>その他</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="webBrowserEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectWebBrowserButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>選択...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>62</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>useCustomWebBrowserRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>webBrowserEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>213</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel" >
     <x>348</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomWebBrowserRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectWebBrowserButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>232</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>469</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
